portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase

```diff
@@ -1010,48 +1010,160 @@ definition {
 	}
 
 	@description = "LPS-99421: The test is removed when the delete action is clicked"
-	@ignore = "true"
 	@priority = "3"
 	test DeleteABTest {
+		task ("Create AB test and a variant") {
+			ABTest.createABTest(
+				goal = "Bounce Rate",
+				testTitle = "AB test title");
 
-		// TODO LPS-144027 DeleteABTest pending implementation
+			ABTest.createVariant(variantName = "Variant name");
+		}
 
+		task ("Delete AB Test") {
+			ABTest.editOrDeleteABTest(value = "Delete");
+		}
+
+		task ("Assert that does not exist ab test") {
+			AssertElementPresent(locator1 = "ABTest#CREATE_TEST_BUTTON");
+		}
 	}
 
 	@description = "LPS-99421: The test is not removed when the delete action is canceled"
-	@ignore = "true"
 	@priority = "3"
 	test DeleteABTestCancel {
+		task ("Create AB test and a variant") {
+			ABTest.createABTest(
+				goal = "Bounce Rate",
+				testTitle = "AB test title");
+
+			ABTest.createVariant(variantName = "Variant name");
+		}
+
+		task ("try to Delete AB Test and click cancel button") {
+			Click(locator1 = "ABTest#KEBAB_AB_TEST");
+
+			ClickNoError(
+				locator1 = "ABTest#KEBAB_OPTIONS_AB_TEST",
+				value = "Delete");
+		}
 
-		// TODO LPS-144027 DeleteABTestCancel pending implementation
+		Confirm.dismissAlert();
```

Comment:
```suggestion
Button.clickCancel
```
Group in task ("try to Delete AB Test and click cancel button") block

Reply:
did not work

Reply:
My bad. I regarded it as delete test in AC